hack/generated/pkg/genruntime/resource_hierarchy.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package genruntime
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type ResourceHierarchyRoot string
```

Comment:
This file was just copied -- not sure why diff makes it show all new

Reply:
Oh, doh -- it didn't exist before and I split it out of the resolver file to make things cleaner. No actual change to the content though.